index.html

```diff
@@ -1391,6 +1402,16 @@
           used to probe information about which <a>remote playback device</a>
           the user has without user consent.
         </p>
+        <p>
+          If a user <a data-lt="disable remote playback">disables remote
+          playback</a> on all <a data-lt="media element">media elements</a>
+          (through a browser setting, for example), the user agent should
```

Comment:
I'm not sure about the use of should and will here. If we're defining the behavior in the response of a user setting, MUST would be preferred (by the definition of the user agent). However, we don't spec browser settings here and it's just a note.

Maybe something like "The user agent MAY at any time stop monitoring the list of available remote playback devices if the user disables remote playback (for example through a browser setting). In this case the availability callbacks MUST be fired with ```false``` and their ids will remain valid however they MAY never be fired again." is more focused on the API?